tee/prototype/epoch.go

```diff
@@ -50,6 +51,14 @@ func cloneBalances(a map[common.Address]*Bal) map[common.Address]*Bal {
 	return b
 }
 
+func (e *Epoch) applyExits(exiters exitersSet) {
+	log.Debugf("Epoch %d: removing %d exiters", e.Number, len(exiters))
```

Comment:
better first do `log := log.WithField("epoch", e.Number)` and them omit the "Epoch %d:" in the next two log calls. We do it like this in (most) other places.

Reply:
Ouh, nice feature.